app/pages/cif/project-revision/[projectRevision]/form/[formIndex]/index.tsx

```diff
@@ -90,7 +99,18 @@ export function ProjectFormPage({
     mode === "view"
   );
 
-  const formPages = useFormPages();
+  const isRedirectingNoFundingStream = useRedirectTo404IfFalsy(
+    query.projectRevision?.projectFormChange?.asProject
+      ?.fundingStreamRfpByFundingStreamRfpId?.fundingStreamByFundingStreamId
+      ?.name
+  );
+
+  const fundingStream = isRedirectingNoFundingStream
+    ? "err"
```

Comment:
In that case, neither funding stream specific items will be shown in the task list because it doesn't know what funding stream it falls under. It shouldn't get this far but it also won't crash on the user